test/test_lint.py

```diff
@@ -2,8 +2,11 @@
 
 import pytest
 from rpmlint.lint import Lint
+from rpmlint.spellcheck import ENCHANT
 
-from Testing import TEST_CONFIG, testpath
+from Testing import (
+    TEST_CONFIG, testpath, HAS_CHECKBASHISMS, HAS_DASH, HAS_RPMDB
```

Comment:
There's a flake8 issue for this:
`test_lint.py:7:1: I101 Imported names are in the wrong order. Should be HAS_CHECKBASHISMS, HAS_DASH, HAS_RPMDB, TEST_CONFIG, testpath`

Reply:
Not sure why my flake version doesn't report this:
```sh
$ flake8 --version
3.9.2 (mccabe: 0.6.1, pycodestyle: 2.7.0, pyflakes: 2.3.1) CPython 3.9.5 on Linux
```
Anyways, it should be fixed now.

Reply:
One need flake8 plugins are listed in Github Action workflow:
https://github.com/rpm-software-management/rpmlint/blob/2402248b1a2f4c77048a61d991537c9c0980b3f0/.github/workflows/main.yml#L39-L45

Reply:
Okay, that explains it. Wouldn't it be a good idea to use `poetry` to easily set up a consistent test environment both locally and in the CI?

Reply:
That would only work for Python packages though.. But it would make the environment easily reproducible for those.

Reply:
> Okay, that explains it. Wouldn't it be a good idea to use `poetry` to easily set up a consistent test environment both locally and in the CI?

I don't know the tool. What is it about? Do you have a link, please?

Reply:
Poetry is not very useful since we _need_ system Python module dependencies.

Reply:
> I don't know the tool. What is it about? Do you have a link, please?

Here: https://github.com/python-poetry/poetry

It creates a virtual environment via Pip, so it could be used to set up a development/test environment making sure that everybody (including CI) have the same dependencies with the same versions.

> Poetry is not very useful since we need system Python module dependencies.

Sure, but I think it could be useful for development & test dependencies. Those do not strictly have to be system dependencies, right? If you run the tests on install it might be necessary though.. Never mind.

Reply:
We need `rpm` and a few others that are not published to PyPI and are only built as part of other programs. Without them, the tests don't work. So Poetry is not useful for testing either.

Reply:
> everybody (including CI) have the same dependencies with the same versions

We are _explicitly_ not doing this because it causes major problems when trying to integrate into distributions.

Reply:
> We are explicitly not doing this because it causes major problems when trying to integrate into distributions.

Right, fully understand that!